src/pages/Admin/ToolsPage/components/Tool/Tool.tsx

```diff
@@ -43,26 +42,33 @@ export const Tool: React.FC<ToolOptions> = ({ tool, worldSlug }) => {
   );
 
   if (returnedData) {
-    return <tool.outputComponent {...returnedData} />;
+    return (
+      <ThreeColumnLayout>
+        <tool.outputComponent {...returnedData} />
```

Comment:
[Q] Does this name look OK?

Reply:
the name of the component is taken from the config of the tool. We can rename it if you have a better name in mind.

```ts
export type SelfServeScript = {
  name: string;
  description: string;
  functionLocation: string;
  arguments: ScriptArgument[];
  isBeta?: boolean;
  // NOTE: We want this component to be trully generic
  // eslint-disable-next-line @typescript-eslint/no-explicit-any
  outputComponent: React.FC<any>;
};
```